### PR TITLE
Adding Lowe's case study

### PR DESCRIPTION
**Pull request by rachelandrew**
This adds the Lowe's site speed case study.

As the content was approved I have not made major changes other than to copy edit. I did break up the big paragraph under the impact heading as it was a bit of a wall of words.

The line at the top with the authors, I have left as is. I'm assuming we don't add each as an author being external?

cc: @mihajlija @garimamimani

### CODE REVIEW

**Comment by mihajlija** [comment]
@rachelandrew We do create author profiles for external contributors, here's an example: https://deploy-preview-4938--web-dev-staging.netlify.app/authors/alonko/ and Garima is a Googler so she should already have an author profile

**Comment by rachelandrew** [comment]
@mihajlija ok cool, I'll do that.

File: src/site/content/en/blog/lowes/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/U0Xrfy2kVLEb3JSiPS5C.png
+alt: The Lowes logo.
+tags:
+  - blog
+  - case-study
+  - web-vitals
+  - performance
+---
+
+This post was authored by [Ashish Choudhury](https://www.linkedin.com/in/choudhuryashish/),
```
Comment (mihajlija) [inline src/site/content/en/blog/lowes/index.md:21]: We do still want to keep this note to highlight that the authors are from Lowe's.

File: src/site/content/en/blog/lowes/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+[Dinakar Chandolu](https://www.linkedin.com/in/dinakarchandolu/),
+[Abhimanyu Raibahadur](https://www.linkedin.com/in/abhimanyuraibahadur/),
+[Dhilipvenkatesh Uvarajan](https://www.linkedin.com/in/dhilip-venkatesh-uvarajan-16914624/),
+[Safwan Samla](https://www.linkedin.com/in/safwansamla/),
+[Garima Mimani](https://www.linkedin.com/in/garimamimani/).
```
Comment (mihajlija) [inline src/site/content/en/blog/lowes/index.md:33]:
```suggestion
This post was authored by [Ashish Choudhury](https://www.linkedin.com/in/choudhuryashish/),
[Dinakar Chandolu](https://www.linkedin.com/in/dinakarchandolu/),
[Abhimanyu Raibahadur](https://www.linkedin.com/in/abhimanyuraibahadur/),
[Dhilipvenkatesh Uvarajan](https://www.linkedin.com/in/dhilip-venkatesh-uvarajan-16914624/),
[Safwan Samla](https://www.linkedin.com/in/safwansamla/),
[Garima Mimani](https://www.linkedin.com/in/garimamimani/).
```
```suggestion
{% Aside %}
This post was authored by [Ashish Choudhury](https://www.linkedin.com/in/choudhuryashish/),
[Dinakar Chandolu](https://www.linkedin.com/in/dinakarchandolu/),
[Abhimanyu Raibahadur](https://www.linkedin.com/in/abhimanyuraibahadur/),
[Dhilipvenkatesh Uvarajan](https://www.linkedin.com/in/dhilip-venkatesh-uvarajan-16914624/),
[Safwan Samla](https://www.linkedin.com/in/safwansamla/) from Lowe's.
{% endAside %}
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/lowes/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  the Lowe's Site Speed Team test pull requests against performance budgets,
+  and prevent performance regressions going into production.
+authors:
+  - choudhuryashish
+  - dinakarchandolu
+  - abhimanyuraibahadur
+  - dhilipvenkateshuvarajan
+  - safwans
+  - gmimani
+date: 2021-03-22
```
Comment (mihajlija) [inline src/site/content/en/blog/lowes/index.md:18]:
```suggestion
date: 2021-03-23
```

**Pull request merged**